DESCRIPTION

```diff
@@ -20,6 +20,7 @@ Imports:
 Suggests:
     DiagrammeR,
     devtools,
+    gtable,
```

Comment:
The ggplot test didn't like my local package configuration. Will remove the dependency again.